nominatim-ui/riscv64.patch

```diff
@@ -0,0 +1,28 @@
+--- PKGBUILD
++++ PKGBUILD
+@@ -14,13 +14,15 @@ backup=('etc/webapps/nominatim-ui/config.theme.js')
+ install=nominatim-ui.install
+ source=("${pkgname}-${pkgver}.tar.gz::${url}/archive/tags/v${pkgver}.tar.gz"
+         'nginx.conf'
+-)
++	'longer-timeout.patch')
+ sha256sums=('31f793bafda4a3259e9b1a4be0cb4dd1a70d856c2fd96db760cbbf64462f8cd6'
+-            '045dda10d978acedba43b071271731b2b63a8ad2743cdd6eda7dab2fd98a52ae')
++            '045dda10d978acedba43b071271731b2b63a8ad2743cdd6eda7dab2fd98a52ae'
++            '914520cf86ec3c212b798a4be3b3619ed1ae12f6ae8746b617069709bb83ceee')
+ 
+ prepare() {
+   cd "${pkgname}-tags-v${pkgver}"
+ 
++  patch -p0 -i ../longer-timeout.patch
+   yarn install
+ }
+ 
+@@ -33,6 +35,7 @@ build() {
+ check() {
+   cd "${pkgname}-tags-v${pkgver}"
+ 
++  export PUPPETEER_EXECUTABLE_PATH="/usr/bin/chromium"
```

Comment:
Wondering why x86_64 don't need to manually set this. Some architecture specific config?

Reply:
Because of puppeteer, the library they used to launch chrome, it will download chrome from google which was not provided RISC-V prebuilt binaries.

Reply:
Please refer their docs: https://pptr.dev/guides/configuration  
In the first paragraph in this page.

Reply:
> Because of puppeteer, the library they used to launch chrome, it will download chrome from google which was not provided RISC-V prebuilt binaries.

Sounds like a bug for Arch Linux, as it declares `checkdepends=('chromium')` but chromium is not used really.  The `PUPPETEER_EXECUTABLE_PATH` setting might be used.

Reply:
This line was merged into archlinux.